composer.json

```diff
@@ -106,7 +106,9 @@
 	},
 	"scripts": {
 		"test": [
```

Comment:
sounds strange, that a test target actually does not run tests

Reply:
Fixed.